fpga_interchange/examples/devices/xc7a35t/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+generate_xc7_device_db(
+    device xc7a35t
+    part xc7a35tcsg324-1
+    device_target target
```

Comment:
Please name the output variable better than `target`.  Maybe `xc7a35t_device`?

Reply:
Sure, done